frontend/src/components/utils/Search.tsx

```diff
@@ -79,7 +87,7 @@ class Search extends Component {
   render() {
     return (
       <form
-        onSubmit={this.handleSearch}
+        onSubmit={(e) => this.handleSearch(e)}
```

Comment:
Hmm, this way we will have a new function-handler on every render. Why do we need to wrap the handler?

Reply:
Yep, probably I tried to figure out things and forgot to clean up :)